src/modules/dashboard/components/ActionsPage/ActionsComponents/MintTokenMotion.tsx

```diff
@@ -186,9 +222,28 @@ const MintTokenMotion = ({
           <CountDownTimer
             createdAt={actionCreatedAt}
             colonyAddress={colony.colonyAddress}
-            text={MSG.stake}
+            text={motionCountdownTimerMsg.stake}
             periodType="stakePeriod"
           />
+          {motionState === MotionState.Voting && votingStateData && (
+            <>
+              <span className={motionSpecificStyles.text}>
+                <FormattedMessage {...MSG.or} />
+              </span>
+              <div className={motionSpecificStyles.progressBarContainer}>
+                <ProgressBar
+                  value={currentReputationPercent}
+                  threshold={threasholdPercent}
+                  max={100}
+                  appearance={{
+                    size: 'small',
+                    backgroundTheme: 'dark',
+                    barTheme: 'danger',
```

Comment:
So the way that I understood the specs, is that the progress bar is green _(primary)_, and the threshold bar red, not both red.

But I might be wrong, so check with Product